moonraker/components/history.py

```diff
@@ -170,7 +171,20 @@ async def _handle_jobs_list(self,
 
     async def _handle_job_totals(self,
                                  web_request: WebRequest
-                                 ) -> Dict[str, Dict[str, float]]:
+                                 ) -> Dict[str, Union[bool, Dict[str, float]]]:
+        action = web_request.get_action()
+        if action == "DELETE":
+            for x in self.job_totals:
+                if isinstance(self.job_totals[x], float):
+                    self.job_totals[x] = float(0.)
+                else:
+                    self.job_totals[x] = int(0)
```

Comment:
I think rather than use a for loop here it might be more clear to re-initialize self.job_totals, ie:
```python
self.job_totals = {
    'total_jobs': 0,
    'total_time': 0.,
    'total_print_time': 0.,
    'total_filament_used': 0.,
    'longest_job': 0.,
    'longest_print': 0.
}
```